### ISSUE TITLE
[2.4.x] class not found for Slf4jLoggingFilter when starting play apps

### ISSUE THREAD

**Issue by hepin1989** [playframework/playframework#5120]
play version is 2.4.3

this Slf4jLoggingFilter is not exists in the akka 2.3 release.
https://github.com/akka/akka/blob/releasing-2.3.14/akka-slf4j/src/main/scala/akka/event/slf4j/Slf4jLogger.scala

but here it was referenced :
https://github.com/playframework/playframework/blob/2.4.x/framework/src/play/src/main/resources/play/reference-overrides.conf

It only lives in akka's master I think.
https://github.com/akka/akka/commit/813543e8f89894cdf14e918bd3e818608fbb9741

after I upgrade the akka-slf4j to akka2.4-rc3 it works,is there any problem?

**Comment by mirkof** [comment]
Well, one solution is to upgrade to akka 2.4. and in conf file reference

```
akka {
  loggers = ["akka.event.slf4j.Slf4jLogger"]
  loglevel = DEBUG
  logging-filter = "akka.event.slf4j.Slf4jLoggingFilter"
}
```

**Comment by hepin1989** [comment]
@mirkof yes,it's a little annoying,thank you anyway.

**Comment by mirkof** [comment]
np :)

**Comment by kuc** [comment]
Snippet for updating `build.sbt`:

```
libraryDependencies ++= Seq(
  // (...)
  "com.typesafe.akka" %% "akka-slf4j" % "2.4.0"
)
```

to make tests working again on Play 2.4.3 and Akka 2.4.0

**Comment by wsargent** [comment]
So the fix here for 2.4.x is to remove Slf4jLoggingFilter from reference-overrides.conf ?

**Comment by richdougherty** [comment]
OK so it looks like support for the `akka.logging-filter` setting was added in Akka 2.4, but we have the setting in Play 2.4, which uses Akka 2.3. Does this cause a bug or is it just something that's untidy?

**Comment by hepin1989** [comment]
yes,upgrade to `akka-slf4j`,at least it works for me.

**Comment by benmccann** [comment]
This should now be fixed in master

**Comment by benmccann** [comment]
Do we need to fix something in 2.4.x still? How does this issue manifest?

**Comment by wsargent** [comment]
Since there's a workaround and this is relatively minor, closing.
